cmd/hostd/main.go

```diff
@@ -104,20 +102,8 @@ func tryLoadConfig() {
 		configPath = str
 	}
 
-	// If the config file doesn't exist, don't try to load it.
-	if _, err := os.Stat(configPath); os.IsNotExist(err) {
-		return
-	}
-
-	f, err := os.Open(configPath)
-	checkFatalError("failed to open config file", err)
-	defer f.Close()
-
-	dec := yaml.NewDecoder(f)
-	dec.KnownFields(true)
-
-	if err := dec.Decode(&cfg); err != nil {
-		fmt.Println("failed to decode config file:", err)
+	if err := config.LoadFile(configPath, &cfg); err != nil && !errors.Is(err, os.ErrNotExist) {
+		fmt.Println("failed to load config file:", err)
```

Comment:
I remember reviewing a `hostd` PR that logged startup errors to stderr. I think failing to load config might be an inconsistent that's either sneaking back in or was overlooked at the time.